ripple/src/main/webapp/script.js

```diff
@@ -138,8 +138,9 @@ function searchInput(keyPress) {
 }
 
 /* Given a formatted user string, converts it to uppercase and removes special characters. */
-function convertToRawString(string) {
-  if ()
+function convertToRawString(str) {
+  var strLower = str.toUpperCase();
```

Comment:
variable name is kind of confusing since its called strLower but everything is uppercase

Reply:
fixed!